evap/templates/base.html

```diff
@@ -160,6 +160,36 @@
         {% endblock %}
     </div>
 </div>
+
+<button id="feedback_button" type="button" class="btn btn-warning" data-toggle="modal" data-target="#feedback_modal">?</button>
```

Comment:
I redesigned all the UI stuff here https://github.com/janno42/EvaP/commit/323f776c010b64a78487c639dba9593892fa4d87, can you please copy it from there?